Navigation/Profile/ProfileHeaderView.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  ProfileHeaderView.swift
+//  Navigation
+//
+//  Created by Admin on 6/9/21.
+//  Copyright © 2021 Artem Novichkov. All rights reserved.
+//
+
+import UIKit
+
+class ProfileHeaderView: UIView {
+
+    let statusUILabel = UILabel()
+    @IBOutlet weak var avatarImageView: UIImageView!{
+        didSet{
+            setupImageView()
+        }
+    }
+    
+
+    
+    @IBOutlet weak var fullNameLabel: UILabel!{
+        didSet{
+            setupFullNameLabel()
+        }
+    }
+    
+    @IBOutlet weak var statusLabel: UILabel!{
+        didSet{
+            setupStatusLabel()
+        }
+    }
+    
+    @IBOutlet weak var setStatusButton: UIButton!{
+        didSet{
+            setupSetStatusButton()
+        }
+    }
+    
+    
+        override init(frame: CGRect) {
+            super.init(frame: frame)
+        }
+        
+        required init?(coder: NSCoder) {
+            super.init(coder: coder)
+            setupNewButton()
+        }
+        
+    func setupImageView(){
+        addSubview(avatarImageView)
+        let imageName = "cat.png"
+        let image = UIImage(named: imageName)
+        avatarImageView.image = image
+        avatarImageView.layer.borderColor = UIColor.white.cgColor
+        avatarImageView.layer.borderWidth = 3
+        avatarImageView.layer.cornerRadius = 50
+        avatarImageView.isUserInteractionEnabled = true
+        let tapRecognizer = UITapGestureRecognizer(target: self, action: #selector(tapAction))
+        avatarImageView.addGestureRecognizer(tapRecognizer)
+        
+    }
+    
```

Comment:
`@objc func tapAction() {
   guard let imageToPresent = avatarImage.image else { 
       print("theirs some thing wrong, image should exist")
       return 
   }
   presenter.present(image: imageToPresent)
}`